1_fetch/src/get_nwis_data.R

```diff
@@ -34,6 +46,9 @@ nwis_site_info <- function(fileout, data){
   site_no <- unique(data$site_no)
   site_info <- dataRetrieval::readNWISsite(site_no)
```

Comment:
This is the one requested change which still needs to be fully implemented. You have changed the function name to match the fact that this is now an object target 👍 

But - now that it is no longer a file target it should no longer be writing a file. So the `fileout` argument needs to be removed from this. Otherwise people reading the `_targets.R` to understand your pipeline could easily think that this is a file target!

Reply:
Ahh, yes! This is another case where I want the target to be object but also want to keep the write_csv() element in there and keep that as a functionality of the entire function. I am implicitly gripping to this idea that this function is originally built in such way, and I must keep as much of it's original parts as possible. Cause again, pipeline works regardless.
--> will get work on this this thought process 😄 

Fixing and pushing!

Reply:
Beautiful! Using targets is a different way of thinking about code for sure - it remaps the mental landscape for all of us! 😁